### ISSUE TITLE
Usage examples

### ISSUE THREAD

**Issue by kulkravi** [jmccormick2001/rqlite-operator#1]
Hello,

I am fairly new to k8s operators. Are there any usage examples for rqlite-operator?

Regards,

Ravi Kulkarni

**Comment by jmccormick2001** [comment]
Hi Ravi,

I was building the rqlite-operator as a way to run rqlite within a k8s cluster similar to other database operators.  I've not worked on it for awhile but the idea was for this operator to setup/provision an rqlite cluster mostly as its main purpose.  The project needs to be updated to the most recent copy of the operator-sdk or kubebuilder as well as the latest version of rqlite.

**Comment by crabique** [comment]
So it's abandoned? Too bad, sounds like a cool project especially as more things begin to implement SQLite support.

**Comment by jmccormick2001** [comment]
yep, I got busy and dropped it....I might pick it up again someday.
